iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -4539,9 +4539,10 @@ struct NativeECSqlStatement : BeObjectWrap<NativeECSqlStatement> {
 
         REQUIRE_ARGUMENT_STRING(1, ecsql);
         OPTIONAL_ARGUMENT_BOOL(2,logErrors, true);
+        OPTIONAL_ARGUMENT_BOOL(3,persistent, false);
         IssueListener listener(*ecdb);
 
-        ECSqlStatus status = m_stmt.Prepare(*ecdb, ecsql.c_str(), logErrors);
+        ECSqlStatus status = m_stmt.Prepare(*ecdb, ecsql.c_str(), logErrors, persistent ? DbPrepareOptions::Persistent : DbPrepareOptions::None);

```

Comment:
I was wondering here, in native we have the DbPrepareOptions (Persistent is the only option), but to typescript we expose just a boolean flag.
Why not do the same thing on both sides, it seems more intuitive. Do we expect there to be more options in the future? In that case an Options object seems correct. But if persistent remains the only option, a boolean flag seems better.